lightning: add parameters to make it more stable to import large mount of data (#13562)
pingcap/docs: tidb-lightning/tidb-lightning-physical-import-mode.md

## Implementation

1. Before importing data, TiDB Lightning automatically switches the TiKV nodes to "import mode", which improves write performance and stops auto-compaction. TiDB Lightning determines whether to pause global scheduling according to the TiDB Lightning version.

    - Starting from v7.1.0, you can you can control the scope of pausing scheduling by using the TiDB Lightning parameter [`pause-pd-scheduler-scope`](/tidb-lightning/tidb-lightning-configuration.md).
    - For TiDB Lightning versions between v6.2.0 and v7.0.0, the behavior of pausing global scheduling depends on the TiDB cluster version. When the TiDB cluster >= v6.1.0, TiDB Lightning pauses scheduling for the Region that stores the target table data. After the import is completed, TiDB Lightning recovers scheduling. For other versions, TiDB Lightning pauses global scheduling.

>     - When TiDB Lightning < v6.2.0 or TiDB cluster < v6.1.0, TiDB Lightning pauses global scheduling.

Reviewer: ```suggestion
    - When TiDB Lightning < v6.2.0, TiDB Lightning pauses global scheduling.
```

## Implementation

1. Before importing data, TiDB Lightning automatically switches the TiKV nodes to "import mode", which improves write performance and stops auto-compaction. TiDB Lightning determines whether to pause global scheduling according to the TiDB Lightning version.

    - Starting from v7.1.0, you can you can control the scope of pausing scheduling by using the TiDB Lightning parameter [`pause-pd-scheduler-scope`](/tidb-lightning/tidb-lightning-configuration.md).
    - For TiDB Lightning versions between v6.2.0 and v7.0.0, the behavior of pausing global scheduling depends on the TiDB cluster version. When the TiDB cluster >= v6.1.0, TiDB Lightning pauses scheduling for the Region that stores the target table data. After the import is completed, TiDB Lightning recovers scheduling. For other versions, TiDB Lightning pauses global scheduling.
    - When TiDB Lightning < v6.2.0 or TiDB cluster < v6.1.0, TiDB Lightning pauses global scheduling.

>     - When TiDB Lightning <= v6.1.0, TiDB Lightning stops global scheduling.

Reviewer: ```suggestion
    - When TiDB Lightning <= v6.1.0, TiDB Lightning pauses global scheduling.
```

>     - When TiDB Lightning <= v6.1.0, TiDB Lightning pauses global scheduling.

Author: ```suggestion
```

> 1. Before importing data, TiDB Lightning automatically switches the TiKV nodes to "import mode", which improves write performance and stops auto-compaction. TiDB Lightning determines whether to pause global scheduling according to the TiDB cluster version.

Reviewer: ```suggestion
1. Before importing data, TiDB Lightning automatically switches the TiKV nodes to "import mode", which improves write performance and stops auto-compaction. TiDB Lightning determines whether to pause global scheduling according to the TiDB Lightning version.
```

>     - Starting from v7.1.0, you can use the TiDB Lightning parameter [pause-pd-scheduler-scope](/tidb-lightning/tidb-lightning-configuration.md) to control the scope of pausing scheduling.
>     - For versions v6.2.0 ~ v7.0.0, TiDB Lightning determines whether to stop global scheduling depending on the version of the TiDB cluster. When the TiDB cluster >= v6.1.0 and TiDB Lightning >= v6.2.0, TiDB Lightning pauses scheduling for the region that stores the target table data. After the import is completed, TiDB Lightning recovers scheduling. For other versions, TiDB Lightning stops global scheduling.
>     - When the TiDB cluster < v6.1.0 or TiDB Lightning < v6.2.0, TiDB Lightning pauses global scheduling.
>     - When TiDB Lightning <= v6.1.0, TiDB Lightning stops global scheduling.

Reviewer: ```suggestion
    - Starting from v7.1.0, you can you can control the scope of pausing scheduling by using the TiDB Lightning parameter [`pause-pd-scheduler-scope`](/tidb-lightning/tidb-lightning-configuration.md).
    - For TiDB Lightning versions between v6.2.0 and v7.0.0, the behavior of pausing global scheduling depends on the TiDB cluster version. When the TiDB cluster >= v6.1.0, TiDB Lightning pauses scheduling for the Region that stores the target table data. After the import is completed, TiDB Lightning recovers scheduling. For other versions, TiDB Lightning pauses global scheduling.
    - When TiDB Lightning < v6.2.0 or TiDB cluster < v6.1.0, TiDB Lightning pauses global scheduling.
```

### Limitations

> - Do not use the physical import mode to directly import data to TiDB clusters in production. It has severe performance implications. If you need to do so, refer to [Pause scheduling on the table level](/tidb-lightning/tidb-lightning-physical-import-mode-usage.md#scope-for-pausing-scheduling-during-import).

Reviewer: ```suggestion
- Do not use the physical import mode to directly import data to TiDB clusters in production. It has severe performance implications. If you need to do so, refer to [Pause scheduling on the table level](/tidb-lightning/tidb-lightning-physical-import-mode-usage.md#scope-of-pausing-scheduling-during-import).
```
